minimal/src/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import '../src/styles/index.css';
+import App from '../src/components/App';
+import { ApolloProvider } from 'react-apollo';
+import { HttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { ApolloClient } from 'apollo-client';
+
```

Comment:
In addition to the above comment, if you are using `apollo-client-preset` you can import `HttpLink`, `InMemoryCache`, and `ApolloClient` from it like:

```js
import { HttpClient, InMemoryCache, ApolloClient } from 'apollo-client-preset'
```